src/core/usecases/launcher.ts

```diff
@@ -518,11 +527,16 @@ export const thunks = {
         (params: {
             catalogId: string;
             packageName: string;
+            version: string | undefined;
             formFieldsValueDifferentFromDefault: FormFieldValue[];
         }): ThunkAction =>
         async (...args) => {
-            const { catalogId, packageName, formFieldsValueDifferentFromDefault } =
-                params;
+            const {
+                catalogId,
+                packageName,
+                "version": specificVersion,
```

Comment:
Should either be named `props_version` (more generic) or `versionSpecifiedAsParam`